src/Sylius/Bundle/AttributeBundle/Resources/config/validation/AttributeTranslation.xml

```diff
@@ -26,5 +26,15 @@
                 <option name="groups">sylius</option>
             </constraint>
         </property>
+        <property name="locale">
```

Comment:
Do we need to check, that locale is exists in database? Or is it checked somewhere else?

Reply:
If I am not wrong, this is not checked anywhere, and probably should be, we will add it in a separate PR